.release/security-scan.hcl

```diff
@@ -0,0 +1,13 @@
+container {
+	dependencies = true
+	alpine_secdb = true
+	secrets      = true
```

Comment:
Is this supposed to be a `block` type? I'm looking at the config docs [here](https://github.com/hashicorp/security-scanner/blob/main/CONFIG.md#block-attributes-1)

Reply:
Yes, this is supposed to be a `block` type. This is the configuration that has been set up for CRT (Common Release Tooling). I'm not sure if I quite answered your question here, but please add follow-up questions and I'd be happy to clarify any bits!

Reply:
It not being a `block` type must not be causing any issues given that all the other projects using the scanner, but I'm curious if this means that it's the same as:
```
secrets {
    all = true
}
```

Ok, I went down the rabbit hole. Apparently, `secrets = true` is a deprecated configuration option, but [it is still supported](https://github.com/hashicorp/security-scanner/blob/feb4c96ac64d9d08fd837f0d11385c234d0b11b1/cmd/scan/detect_config.go#L67-L88). Using `secrets = true` is the same as `secrets { all = true }`.

I wonder if we should be using the updated `block` type here to future-proof it?

Reply:
Update has been made!